presto-tpch/src/main/java/com/facebook/presto/tpch/TpchRecordSet.java

```diff
@@ -67,13 +67,15 @@
     private final List<Type> columnTypes;
     private final Optional<TupleDomain<ColumnHandle>> predicate;
 
-    public TpchRecordSet(Iterable<E> rows, TpchTable<E> table, Iterable<TpchColumn<E>> columns, Optional<TupleDomain<ColumnHandle>> predicate)
```

Comment:
Why this change?

Reply:
Because I didn't like: `ImmutableList.copyOf(transform(columns, TpchMetadata::getPrestoType))` below. And because of that nobody really cared that is `Iterable`, so there is no need to complicate things.